apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/__snapshots__/index-table.spec.tsx.snap

```diff
@@ -41,7 +41,7 @@ exports[` 1`] = `
               aria-invalid="false"
```

Comment:
Do we really need this test given that we have good Cypress test coverage now?

Reply:
It's just a unit test of the implementation. But if it continues to fail due to refactors, we should remove this test.